src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Options/ProcessorOptions.cs

```diff
@@ -0,0 +1,12 @@
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Options;
+
+public class ProcessorOptions
+{
+    public static readonly string Section = "Processor";
```

Comment:
Not too sure about this name. We're in the `Public.Data.Processor` project, so everything is implicitly a processor option.

For the `Data.Processor`, we've called a similar thing `AppSettings`. Maybe it'd be better to be consistent with this? Or is `AppSettings` a bit weird and we should call it something else?